CHANGELOG.md

```diff
@@ -19,6 +19,11 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* Fix `incompatible encoding regexp match` for linting non-ascii pod name
```

Comment:
last nit: please add two spaces at the end of this sentence for markdown formatting